packages/integrations/src/extraerrordata.ts

```diff
@@ -117,6 +125,12 @@ function _extractErrorData(error: ExtendedError): Record<string, unknown> | null
       extraErrorInfo[key] = isError(value) ? value.toString() : value;
     }
 
+    // Error.cause is a standard property that is non enumerable, we therefore need to access it separately.
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause
+    if (captureErrorCause && error['cause']) {
+      extraErrorInfo['cause'] = isError(error['cause']) ? error['cause'].toString() : error['cause'];
```

Comment:
This would result in the same behavior since `toString()` method is called automatically:

```suggestion
      extraErrorInfo['cause'] = `${error['cause']}`;
```

Reply:
If `error.cause` is an for example an object, this change will be wrong as we want the full object to be on there.